docs/index.html

```diff
@@ -275,23 +310,41 @@ <h2>#TwitterThrowback</h2>
             loadingIndicator.style.display = 'block';
         }, 1000);
     }
+
+    function shareToTwitter() {
+        let url;
+        if(window.username !== undefined) {
+            url = `http://twitter.com/share?text=Here's my TwitterThrowback. Check yours here` +
+                `&url=http://oldtweets.today/?username=${window.username}&hashtags=OldTweetsToday,TwitterThrowback`;
+        }
+        else {
+            url = `http://twitter.com/share?text=I just found out this cool site that helps you see your throwback tweets ` +
```

Comment:
Let's make the text "Wanna see what you were tweeting on this day in the past? Check out {URL}."

Reply:
Done.